index.html

```diff
@@ -0,0 +1,116 @@
+            
+            <title>Portfolio-setup-and-mobile-version-skeleton-Paul Malongo</title>
+        </head>
+<body>
+    <!--Top navigation Bar-->   
+    <header class="mainnav ">
+      <nav class="navt">
+        <i class=" fa fa-signal "></i>
+        <i class=" fa fa-wifi "></i>
+        <i class=" fa fa-battery-full"></i>
+        
+        </nav>
+        <nav class="navtop flame">
+            <div class="Paul">Paul Malongo</div>
+            
+            <!--Hamburge Menu--> 
+            <i class="top fa fa-bars fa-3x"></i>
+            
+        </nav>
+    </header>
+     <!--Main Flexbox Container--> 
+<div class="container flame">
+    <!--Headed Details--> 
+    <div class="Mname"><h1>Paul Malongo</h1></div>
+    <div class="SE"><h3>Software developer</h3></div>
+    <!--Introduction--> 
+    <div > 
+        <p class="detail">I can build a Product, 
+            Feature or Website. Look 
+            through some of my work and 
+            experience! if you like what 
+            you See and have a project you 
+            need coded, dont hesitate to 
+            contact me.
+            
+    
+        </p>
+        <hr>
+    </div>
+        <!--Lists --> 
+        <div class="tbl">
+            
+            <ul>
+              <li class="HL">Language</li>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+    
+            </ul>
+            
+    
+            <ul>
+                <li class="HL">Frameworks</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+      
+              </ul>
+    
+              <ul>
+                <li class="HL">Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+      
+              </ul>
+            </div>
+            <!--Eclipse Elements--> 
+            <div class="eclips"></div>
+            <div class="eclips"></div>
+            <div class="eclip1"></div>
+            <div class="eclips1"></div>
+            <i class="fa-brands fa-angellist"></i>
+
+<!--Social Media Navigators--> 
+
+
+<div class="social">
+
+
+  <a href="https://github.com/">  <i class="fa fa-github fa-3x" aria-hidden="true"></i></a>
+  <a href="https://www.facebook.com/" >  <i class="fa fa-facebook  fa-3x" aria-hidden="true"></i>  </a>
+  <a href="https://www.linkedin.com/">  <i class="fa fa-linkedin-in  fa-3x" aria-hidden="true"></i>  </a>
+  <a href="https://www.angellist.com">  <i class="fa fa-angellist fa-3x" aria-hidden="true"></i>  </a>
+  <a href="https://twitter.com/i/flow/login">  <i class="fa fa-twitter fa-3x" aria-hidden="true"></i>  </a>
+  <a href="https://www.instagram.com">  <i class="fa fa-instagram fa-3x" aria-hidden="true"></i>  </a>
```

Comment:
- [ ] Great job aligning your social media icons :clap: . However, it would have been better to include them inside an unordered list i.r `<ul> .. <li></li> .. </ul>` tag. This is a semantically correct approach to aligning your social media icons. For a visual of what I mean kindly consider the following:
```
<ul class="social">
<li>  <a href="https://github.com/">  <i class="fa fa-github fa-3x" aria-hidden="true"></i></a></li>
</ul>
```

- [ ] I have also noticed that you have included icons that are not included in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) you chose. While this is a good effort you made, kindly stick as much as possible to both the Figma design and the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) so you do not have trouble with future reviewers.
- [ ] Also, kindly make sure that your icons are looking like the ones in the chosen Figma design. i.e their size looks very large, it would be better to make them smaller and better appealing. What do you think?

**This is what your Social icons look like right now:**
![image](https://user-images.githubusercontent.com/86318284/176987627-fb177bec-4d08-4a82-a321-812edddc2941.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/176987651-59c4f7d0-e633-4d14-a1f8-9e50a8c62ea0.png)